src/main/java/org/openstreetmap/atlas/generator/dataframe/README.md

```diff
@@ -0,0 +1,29 @@
+# `AtlasDataFrame`
```

Comment:
Can you add a relative link to this page here? https://github.com/osmlab/atlas/blob/dev/README.md#whats-in-it

Reply:
Does the relative link need to be from within the same repo? It seems that all the others are contained within atlas

Reply:
Ah right! Different repo haha! Maybe we need to have it linked in the atlas-generator's main readme?

Reply:
That's probably for another PR though. We need to add more discoverability there..